p2panda-rs/src/identity/key_pair.rs

```diff
@@ -128,11 +128,19 @@ impl KeyPair {
         message: &[u8],
         signature: &Signature,
     ) -> Result<(), KeyPairError> {
-        public_key.verify(message, &signature)?;
+        public_key.verify(message, signature)?;
         Ok(())
     }
 }
 
+impl Default for KeyPair {
+    fn default() -> Self {
+        let mut csprng: OsRng = OsRng {};
+        let key_pair = Ed25519Keypair::generate(&mut csprng);
+        Self(key_pair)
```

Comment:
Could we just call KeyPair::new() here to avoid duplication?

Reply:
:+1: